crates/astria-bridge-withdrawer/src/config.rs

```diff
@@ -26,6 +26,8 @@ pub struct Config {
     pub rollup_asset_denomination: asset::denom::TracePrefixed,
     // The bridge address corresponding to the bridged rollup asset on the sequencer.
     pub sequencer_bridge_address: String,
+    // Whether to use compat addresses for `Ics20Withdrawal`s.
+    pub use_compat_address: bool,
```

Comment:
We have a one-withdrawer-per-asset (and destination) situation, right?

Reply:
yeah exactly, the withdrawer is one instance per bridge/asset, so the withdrawer for noble usdc would set this to true for example